identity/identity.go

```diff
@@ -90,6 +94,9 @@ func (s *StateStorage) All(ops builder.Operations) map[types.NodeID][]StateInfo
 		allEvents[id] = append(allEvents[id], *event)
 		return true
 	})
+	if err != nil {
+		panic(err.Error())
+	}
```

Comment:
Would it be much effort to instead bubble up that error? or at least use `s.logger.Panic` instead?

Reply:
Yes, I forgot about it. Fixed already, thx :+1: